FastSurferCNN/utils/parser_defaults.py

```diff
@@ -90,10 +102,10 @@ def _stub(parser):
         help="Absolute path to file in which a list of subjects that failed QC check  (when processing multiple "
              "subjects) will be saved. If not set, the file will not be saved."),
     "vox_size": __arg(
-        '--vox_size', choices=["auto", "1"], default="auto", dest='vox_size',
-        help="Choose the primary voxelsize to process, must be either '1' or 'auto'. '1' forces "
-             "processing at 1mm voxel size, 'auto' enables processing at higher resolutions (conforming"
-             " to the smallest voxel edge length). Select 'auto' (default) for hires processing."),
+        '--vox_size', type=__vox_size, default="min", dest='vox_size',
```

Comment:
Should this not include a disclaimer that if the res is above 0.95 it will be set to 1.0 mm with min?

Reply:
Well, all that conforming did not actually happen yet :(
So I think I added that now. I'll have to test the code now.